tasks/post_copy_request_to_queue/post_copy_request_to_queue.py

```diff
@@ -10,41 +10,43 @@
 import boto3
 from typing import Dict, Any, List, Optional
 from datetime import datetime, timezone
-import shared_recovery
-import requests_db
-import database
+from shared_recovery import (
+    OrcaStatus,
+    RequestMethod,
+    post_status_for_file_to_queue,
+    post_status_for_job_to_queue,
+)
+from requests_db import get_dbconnect_info
+from database import single_query, result_to_json
 from requests_db import DatabaseError
+from cumulus_logger import CumulusLogger
 
-PREFIX = 'dev'
-DB_HOST = 'dev-drdb-host'
-DATABASE_NAME = 'postgres'
-DATABASE_PORT = '5432'
-DATABASE_USER = 'rhassan'
-OS_ENVIRON_DB_QUEUE_URL_KEY = 'DB_QUEUE_URL'
-OS_ENVIRON_RECOVERY_QUEUE_URL_KEY = 'DB_QUEUE_URL_RECOVERY'
+# instantiate CumulusLogger
+LOGGER = CumulusLogger()
 
+PREFIX = os.environ["PREFIX"]
+DATABASE_NAME = os.environ["DATABASE_NAME"]
+DATABASE_PORT = os.environ["DATABASE_PORT"]
+DATABASE_USER = os.environ["DATABASE_USER"]
+DB_QUEUE_URL = os.environ["DB_QUEUE_URL"]
+RECOVERY_QUEUE_URL = os.environ["RECOVERY_QUEUE_URL"]
 
-def lambda_handler(event, context):
+
+def task(records):
```

Comment:
Add type hints. See the style guide https://google.github.io/styleguide/pyguide.html#s3.8.3-functions-and-methods. You also may need to add more arguments. See the comment about the global variables.